content/account_management/faq/how-do-i-configure-google-as-a-saml-idp.md

```diff
@@ -16,10 +16,12 @@ further_reading:
 
 ## For the "Service Provider Details"
 
+**Pre-requisite**: IDP initiated SSO must be checked on DataDog SAML Configuration page
+
 * **Application Name**: Can be anything
 * **Description**: Can be anything
-* **ACS URL**: use the value shown under "Assertion Consumer Service URL" on https://app.datadoghq.com/saml/saml_setup. (e.g. https://app.datadoghq.com/account/saml/assertion). If there is more than one value shown for Assertion Consumer Service URL, only enter one of them here.
-* **Entity ID**: https://app.datadoghq.com/account/saml/metadata.xml
+* **ACS URL**: Use the url shown under "Assertion Consumer Service URL" on https://app.datadoghq.com/saml/saml_setup (the one containing `/id/<COMPAGNY_ID>`). If there is more than one value shown for Assertion Consumer Service URL, only enter one of them here.
```

Comment:
Should it be `COMPAGNY_ID` or `COMPANY_ID`?

Reply:
oops, French way of writing it..